### ISSUE TITLE
WCAG non-compliance

### ISSUE THREAD

**Issue by abelpink** [PEDSnet/PEDSpace#66]
I just ran our homepage through a WCAG audit report generator, and it gave us a pretty bad score. We need to make the following adjustments to make our site more accessible:

1. "ARIA roles sometimes require a parent element with a specific role to function properly. This ensures that elements are structured in a way that assistive technologies can understand the relationships between them. For example, certain ARIA roles like menuitem or treeitem need to be contained within specific parent roles like menu or tree Missing required parent elements can break these relationships, leading to an incomplete or confusing experience for users of screen readers."
![Image](https://github.com/user-attachments/assets/14c81fa9-6dc2-4c02-8993-097a3bb2e585)

2. Not exactly sure what this error is for, but I don't think it likes our buttons.
![Image](https://github.com/user-attachments/assets/c4912d25-ad52-4cf3-b485-c29224a0f877)

3. Our link font needs to have more contrast:
![Image](https://github.com/user-attachments/assets/2fc4b3f2-ff34-403b-a1fe-0c5a2837f0b1)

I think the easiest way to see the suggested improvements is to go to https://www.accessibilitychecker.org/, input the homepage URL and then read the results.

**Comment by aseyedia** [comment]
These issues have to do with the DSpace base frontend itself and not any of the modifications I've made to it. Here's our compliance score:

![image](https://github.com/user-attachments/assets/eea58cf4-9b0f-41ff-9743-41af9c52660c)

Here's [the demo DSpace instance's](demo.dspace.org) compliance:

![image](https://github.com/user-attachments/assets/90dc7888-f23e-487d-8ed0-e2f912fb8a68)

They're basically the same. Working on this would be the job of an open-source developer, unfortunately. Nothing I did to our version of the DSpace instance contributed to the inaccessibility or non-compliance of the site.

**Comment by aseyedia** [comment]
All apologies to @abelpink - I misunderstood the ask. Should be good now. The contrast ratio should be 4.55:1, I shot for 5:1.

![image](https://github.com/user-attachments/assets/826eca53-e381-43b4-a750-29312ddf7ef9)

**Comment by aseyedia** [comment]
![image](https://github.com/user-attachments/assets/bb8288b5-37f2-4dda-959e-e740129586f4)
It's still giving us an issue but for the badges and not for the hyperlinks. I'll work on this more on Friday.

**Comment by aseyedia** [comment]
![image](https://github.com/user-attachments/assets/2d1441e5-ec76-42dd-a7cf-9e44f698ede2)

Fixed all interaction issues.

**Comment by aseyedia** [comment]
You can use [this tool](https://webaim.org/resources/contrastchecker/) to check colors.

**Comment by aseyedia** [comment]
Resolution: roll back the badge colors and instead make all text on badge "Jet" black: `#2A2A2A`
